crates/bevy_window/src/window.rs

```diff
@@ -145,6 +145,40 @@ impl WindowResizeConstraints {
     }
 }
 
+/// Mirror of `Icon` from `winit`.
```

Comment:
Probably link to both. It can't be a doc link, because winit isn't a dependency, but I think a URL is still helpful.

Reply:
[ac4197d](https://github.com/bevyengine/bevy/pull/5488/commits/ac4197d7f89b56c6390b496856279debd901bb63) resolves this.